examples/vertx-opentracing/README.md

```diff
@@ -0,0 +1,153 @@
+
+Make note of the UI address provided by the `oc cluster up` command.
+Once Hawkular APM is up and running, the example as a whole needs to be build and the microservices needs to be added to OpenShift.
+The following commands can be executed in order to accomplish that:
+
+    mvn clean install
+    ## account manager
+    cd account-manager
+    oc new-build --binary --name=account-manager -l app=account-manager
+    oc start-build account-manager --from-dir=. --follow
+    oc new-app account-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=account-manager
+    cd ..
+    ## inventory manager
+    cd inventory-manager
+    oc new-build --binary --name=inventory-manager -l app=inventory-manager
+    oc start-build inventory-manager --from-dir=. --follow
+    oc new-app inventory-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=inventory-manager
+    cd ..
+    ## order log
+    cd order-log
+    oc new-build --binary --name=order-log -l app=order-log
+    oc start-build order-log --from-dir=. --follow
+    oc new-app order-log HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=order-log
+    cd ..
+    ## order manager
+    cd order-manager
+    oc new-build --binary --name=order-manager -l app=order-manager
+    oc start-build order-manager --from-dir=. --follow
+    oc new-app order-manager HAWKULAR_APM_URI="http://hawkular-apm" HAWKULAR_APM_USERNAME="admin" HAWKULAR_APM_PASSWORD="password" -l app=order-manager
+    oc expose service order-manager
+    cd ..
+
+Login to the OpenShift UI at the address given by the `oc cluster up` command and wait until all services are "blue", meaning
+that they are ready. At this point, you can execute the following command to test, replacing the hostname by the appropriate value,
+which can be seen on the OpenShift UI:
+
+    curl -X POST -d '{"accountId":"fred","itemId":"laptop","quantity":2}' http://order-manager-myproject.<your OpenShift IP>.xip.io/orders
+
+The Hawkular APM UI can be accessed via https://hawkular-apm-myproject.<your OpenShift IP>.xip.io/ . The username is `admin` and the
+password is `password`.
+
+## Running it outside of OpenShift
+
+It's possible to run the examples outside of OpenShift as well. Assuming a Hawkular APM server running on `http://localhost:8080`,
+it's necessary to first export the required environment variables:
+
+    export HAWKULAR_APM_URI="http://127.0.0.1:8080"
+    export HAWKULAR_APM_USERNAME="jdoe"
+    export HAWKULAR_APM_PASSWORD="password"
+
+Then, each microservice has to be started, possibly on its own terminal (don't forget to set the env vars above on each terminal).
+From the directory of this README file, run:
+
+    java -jar account-manager/target/hawkular-apm-tests-app-vertx-opentracing-account-manager-fat.jar -cluster
```

Comment:
Could you also add `-DHAWKULAR_APM_SERVICE_NAME=....` to each command, to show setting the service externally?